app/src/pages/playground/PlaygroundExperimentRunDetailsDialog.tsx

```diff
@@ -0,0 +1,212 @@
+                input
+                output
+              }
+            }
+            annotations {
+              edges {
+                annotation: node {
+                  id
+                  name
+                  label
+                  score
+                  explanation
+                  annotatorKind
+                }
+              }
+            }
+          }
+        }
+      }
+    `,
+    { runId }
+  );
+  const run = data.run;
+  const exampleId = run.example?.id;
+  const revision = run.example?.revision;
+  const input = revision?.input;
+  const referenceOutput = revision?.output;
+  return (
+    <Dialog
+      title={`Experiment Run for Example: ${exampleId}`}
+      size="fullscreen"
+    >
+      <PanelGroup direction="vertical" autoSaveId="example-compare-panel-group">
+        <Panel defaultSize={100}>
+          <div
+            css={css`
+              overflow-y: auto;
+              height: 100%;
+            `}
+          >
+            <View overflow="hidden" padding="size-200">
+              <Flex direction="row" gap="size-200" flex="1 1 auto">
+                <View width="50%">
+                  <Card
+                    title="Input"
+                    {...defaultCardProps}
+                    bodyStyle={{
+                      padding: 0,
+                      maxHeight: "300px",
+                      overflowY: "auto",
+                    }}
+                    extra={
+                      <CopyToClipboardButton text={JSON.stringify(input)} />
+                    }
+                  >
+                    <JSONBlock value={JSON.stringify(input, null, 2)} />
+                  </Card>
+                </View>
+                <View width="50%">
+                  <Card
+                    title="Reference Output"
+                    {...defaultCardProps}
+                    extra={
+                      <CopyToClipboardButton
+                        text={JSON.stringify(referenceOutput)}
+                      />
+                    }
+                    bodyStyle={{
+                      padding: 0,
+                      maxHeight: "300px",
+                      overflowY: "auto",
+                    }}
+                  >
+                    <JSONBlock
+                      value={JSON.stringify(referenceOutput, null, 2)}
+                    />
+                  </Card>
+                </View>
+              </Flex>
+            </View>
+          </div>
+        </Panel>
+        <PanelResizeHandle css={resizeHandleCSS} />
+        <Panel defaultSize={200}>
+          <Flex direction="column" height="100%">
+            <View
+              paddingStart="size-200"
+              paddingEnd="size-200"
+              paddingTop="size-100"
+              paddingBottom="size-100"
+              borderBottomColor="dark"
+              borderBottomWidth="thin"
+              flex="none"
+            >
+              <Heading level={2}>Experiment</Heading>
```

Comment:
Does "Experiment Output" make sense or nah? I had to close and open the slideover to confirm I was looking at the same data as the output from the cell I clicked on